lib/pymedphys/_experimental/streamlit/apps/transfer_check.py

```diff
@@ -33,8 +37,14 @@
 TITLE = "Pre-Treatment Data Transfer Check"
 
 
+@st.cache(allow_output_mutation=True, suppress_st_warning=True)
+def get_mosaiq_cursor(server):
+    return uncached_get_mosaiq_cursor(server)
+
+
 def main():
     server = "PRDMOSAIQIWVV01.utmsa.local"
+    cursor = get_mosaiq_cursor(server)
```

Comment:
It's worth opening the Windows Credential Manager (https://support.microsoft.com/en-us/windows/accessing-credential-manager-1b5c916a-6a16-889f-8581-fc16e8165ac0), deleting the old credentials, and seeing if the password input is now done through Streamlit.

Reply:
Just confirmed this indeed works 👍